apps/root/src/pages/earn/components/strategies-table/index.tsx

```diff
@@ -0,0 +1,267 @@
+
+const Row = ({ strategy, onRowClick }: RowProps) => {
+  const [hovered, setHovered] = React.useState(false);
+
+  return (
+    <TableRow
+      key={strategy.id}
+      onMouseEnter={() => setHovered(true)}
+      onMouseLeave={() => setHovered(false)}
+      sx={{ cursor: 'pointer' }}
+      hover
+      onClick={() => onRowClick(strategy)}
+    >
+      {strategyColumnConfigs.map((column) => (
+        <TableCell key={`${strategy.id}-${column.key}`}>{renderBodyCell(column.renderCell(strategy))}</TableCell>
+      ))}
+      <StyledTableEnd size="small">
+        <StyledNavContainer alignItems="center">
+          <DividerBorder2 orientation="vertical" />
+          <AnimatedChevronRightIcon $hovered={hovered} color="primary" />
+        </StyledNavContainer>
+      </StyledTableEnd>
+    </TableRow>
+  );
+};
+
+const createEmptyRows = (rowCount: number) => {
+  return Array.from({ length: rowCount }, (_, i) => (
+    <TableRow key={i} sx={{ visibility: 'hidden', height: ({ spacing }) => spacing(15.25) }}>
+      <TableCell colSpan={7}>&nbsp;</TableCell>
+    </TableRow>
+  ));
+};
+
+const AllStrategiesTable = ({
+  strategies,
+  hasFetchedAllStrategies,
+  defaultOrderBy,
+}: {
+  strategies: TableStrategy[];
+  hasFetchedAllStrategies: boolean;
+  defaultOrderBy: StrategyColumnKeys;
+}) => {
+  const [order, setOrder] = React.useState<Order>('desc');
+  const [orderBy, setOrderBy] = React.useState<StrategyColumnKeys>(defaultOrderBy);
+  const [page, setPage] = React.useState(0);
+  const pushToHistory = usePushToHistory();
+  const trackEvent = useTrackEvent();
+  const dispatch = useAppDispatch();
+
+  const onRowClick = React.useCallback(
+    (strategy: TableStrategy) => {
+      pushToHistory(`/earn/${strategy.network.chainId}/vaults/${strategy.id}`);
+      trackEvent('Earn Vault List - Go to vault details', {
+        chainId: strategy.network.chainId,
+      });
+    },
+    [pushToHistory, trackEvent]
+  );
+
+  const handleSearchChange = debounce((newValue: string) => {
+    setPage(0);
+    dispatch(setSearch(newValue));
+  }, 500);
+
+  const visibleRows = React.useMemo<TableStrategy[]>(
+    () =>
+      strategies
+        .slice()
+        .sort(getComparator(order, orderBy))
+        .slice(page * ROWS_PER_PAGE, page * ROWS_PER_PAGE + ROWS_PER_PAGE),
+    [page, strategies, order, orderBy]
+  );
+
+  // Keeps the table height consistent
+  const emptyRows = createEmptyRows(ROWS_PER_PAGE - visibleRows.length);
+
+  const handleRequestSort = (property: StrategyColumnKeys) => {
+    const isAsc = orderBy === property && order === 'asc';
+    setOrder(isAsc ? 'desc' : 'asc');
+    setOrderBy(property);
+  };
+
+  return (
+    <ContainerBox flexDirection="column" gap={5} flex={1}>
+      <AllStrategiesTableToolbar isLoading={!hasFetchedAllStrategies} handleSearchChange={handleSearchChange} />
+      <StyledBackgroundPaper variant="outlined">
+        <TableContainer component={Paper} elevation={0}>
```

Comment:
so a paper inside a paper, cant we just send StyledBackgroundPaper to this tableContainer component?

Reply:
Removing that paper got involved adding some styles to our tables in general. Now it looks much better in my opinion!